mis_builder/models/mis_report_instance.py

```diff
@@ -478,6 +478,7 @@ def _compute_pivot_date(self):
     _description = "MIS Report Instance"
 
     name = fields.Char(required=True, translate=True)
+    sequence = fields.Integer()
```

Comment:
```suggestion
    sequence = fields.Integer(default=10)
```

The field sequence should have a value by default.
Also it should be in `_order` of the the model.